build.gradle.kts

```diff
@@ -69,48 +73,57 @@ dependencies {
     implementation("org.flywaydb", "flyway-core", "7.8.2")
 }
 
-tasks {
-    val jvmTarget = "1.8"
-    compileKotlin {
-        kotlinOptions.jvmTarget = jvmTarget
-    }
-    compileJava {
-        targetCompatibility = jvmTarget
+ktlint {
+    verbose.set(true)
+    outputToConsole.set(true)
+    coloredOutput.set(true)
+    reporters {
+        reporter(ReporterType.CHECKSTYLE)
+        reporter(ReporterType.JSON)
+        reporter(ReporterType.HTML)
     }
-    compileTestKotlin {
-        kotlinOptions.jvmTarget = jvmTarget
-    }
-    compileTestJava {
-        targetCompatibility = jvmTarget
+    filter {
+        exclude { element ->
+            element.file.path.contains("generated/")
+        }
```

Comment:
Your projects probably does not create a folder /build/generated when building it so you can remove this filter